quic/s2n-quic-core/src/path/mod.rs

```diff
@@ -245,8 +245,12 @@ impl Handle for Tuple {
 
     #[inline]
     fn maybe_update(&mut self, other: &Self) {
-        // once we discover our path, update the address local address
-        if self.local_address.port() == 0 {
+        if other.local_address.port() == 0 {
+            return;
+        }
+
+        // once we discover our path, or the port changes, update the address full address
```

Comment:
I know its just copy-pasted, but `update the address full address` isn't making sense to me grammatically

Reply:
i should have more carefully read it 😄. i'll fix it